client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/plan-only.tsx

```diff
@@ -194,6 +224,39 @@ const PlanOnlyThankYou = ( {
 		},
 	} );
 
+	let upsellProps;
+	if ( promotePlanData ) {
+		upsellProps = {
```

Comment:
We could use the same term everywhere - I think that consistency may help people digging into that code:

```suggestion
	let upsellProps;
	if ( upsellPlanData ) {
		upsellProps = {
```

Reply:
Agree, you are totally right 👍